core/templates/dev/head/domain/classifier/AnswerClassificationResultObjectFactory.ts

```diff
@@ -28,7 +28,7 @@ oppia.factory('AnswerClassificationResultObjectFactory', [function() {
     this.classificationCategorization = classificationCategorization;
   };
 
-  // TODO (ankita240796) Remove the bracket notation once Angular2 gets in.
+  // TODO(ankita240796) Remove the bracket notation once Angular2 gets in.
```

Comment:
I'm not sure whether the pattern is `TODO(userName) <message>` OR `TODO(userName): <message>`. (or both)

Reply:
Good catch! Let's go with `TODO(userName): XXX`. 

@teddymarchildon would that be possible? Sorry for not catching this earlier.

Reply:
Yes! Just updated the regexp to enforce the : as well as a message following the colon.